schema.sql

```diff
@@ -8,3 +8,63 @@ CREATE TABLE animals (
     neutered BOOLEAN,
     weight_kg DECIMAL
 );
+
+-- Table named owners with the required columns
+CREATE TABLE owners (
+    id SERIAL PRIMARY KEY,
+    fullname CHAR(200),
+    age int
+);
+
+-- table named species with the following columns
+CREATE TABLE species (
+    id SERIAL PRIMARY KEY,
+    name CHAR(200)
```

Comment:
- [ Optional] Kindly use `varchar` here as well.

Reply:
Thanks for your review, I will put these suggestions into consideration.